django/contrib/admin/tests.py

```diff
@@ -14,11 +15,41 @@ def process_response(self, request, response):
         return response
 
 
+class AdminSeleniumMetaClass(type):
+    def __new__(cls, name, bases, attrs):
+        """
+        Dynamically inject new methods for running tests in different browsers
+        when multiple browser specs are provided (e.g. --selenium=firefox,chrome).
+        """
+        all_specs = os.environ.get('DJANGO_SELENIUM_SPECS', '').split(',')
+        if all_specs:
+            for key, value in attrs.items():
+                if isinstance(value, types.FunctionType) and key.startswith('test'):
+                    func = value
+                    for index, spec in enumerate(all_specs):
+                        new_func_name = '%s__%s' % (key, spec.replace('/', '_'))
```

Comment:
Some comments in this method would be useful, e.g. why do we need to replace / with _ (I don't think any specs have / in them)?